### DISCUSSION TITLE
Mikrotik - jak użyć list

### DISCUSSION THREAD

**Discussion by ArturNadolski** [MajkiIT/polish-ads-filter#21305] [Q&A]
Cześć,
Widzę całą listę list przygotowanych dla Mikrotika: [polish-ads-filter](https://github.com/MajkiIT/polish-ads-filter)/polish-mikrotik-filters/
Jak tego użyć? Brakuje skryptu który  będzie pobierał, aktualizował i stosował reguły.

**Comment by MajkiIT** [comment]
https://www.certyficate.it/blokowanie-domen-z-listy-holecert-na-mikrotik/ tylko nie wiem czy mikrotik wytrzyma tyle wpisów
